eng/helix/content/InstallNode.ps1

```diff
@@ -0,0 +1,60 @@
+
+param(
+    [Parameter(Mandatory = $true)]
+    $Version,
+    
+    [Parameter(Mandatory = $true)]
+    $output_dir
+)
+
+$ErrorActionPreference = 'Stop'
+$ProgressPreference = 'SilentlyContinue' # Workaround PowerShell/PowerShell#2138
+
+Set-StrictMode -Version 1
+
+if ((Get-Command "node.exe" -ErrorAction SilentlyContinue)) 
+{ 
+    Write-Host "Found node.exe in PATH"
+    exit
+}
+
+if (Test-Path "$output_dir\node.exe")
+{
+    Write-Host "Node.exe found at $output_dir"
+    exit
+}
+
+$nodeFile="node-v$Version-win-x64"
+$url="http://nodejs.org/dist/v$Version/$nodeFile.zip"
+Write-Host "Starting download of NodeJs ${Version} from $url"
+Invoke-WebRequest -UseBasicParsing -Uri "$url" -OutFile "nodejs.zip"
```

Comment:
Retries?

Reply:
Helix automatically retries failed work items 3 times, that seems good enough, I don't think it adds much value adding nested retries here